Controllers/CouriersController.go

```diff
@@ -7,48 +7,41 @@ import (
 	"strconv"
 )
 
-// getCouriers godoc
```

Comment:
This PR shouldn't remove swagger comments. Looks like incorrectly resolved merge conflict